extractor/src/main/java/org/schabi/newpipe/extractor/exceptions/ConsentRequiredException.java

```diff
@@ -0,0 +1,12 @@
+package org.schabi.newpipe.extractor.exceptions;
+
+public class ConsentRequiredException extends ParsingException {
```

Comment:
Maybe it should be a `ContentNotAvailableException`: it is not the extractor's fault if the page can't be loaded, but rather YouTube not providing mixes without consent, so it is a content no available